src/QualificationRunner.Core/Services/QualificationRunner.cs

```diff
@@ -98,23 +97,35 @@ private Task<SaticFiles> copyStaticFiles(dynamic qualificationPlan)
 
       private ObservedDataMapping copyObservedData(ObservedDataMapping observedDataMapping)
       {
-         var observedDataFilePath = Path.Combine(_runOptions.ConfigurationFile, observedDataMapping.Path);
+         var observedDataFilePath = absolutePathFrom(_runOptions.ConfigurationFolder, observedDataMapping.Path);
          var fileInfo = new FileInfo(observedDataFilePath);
          if (!fileInfo.Exists)
             throw new QualificationRunnerException(ObservedDataFileNotFound(observedDataFilePath));
 
          DirectoryHelper.CreateDirectory(_runOptions.ObservedDataFolder);
-         var copiedObservedDataFilePath = Path.Combine(_runOptions.ObservedDataFolder, fileInfo.Name);
+         var copiedObservedDataFilePath = absolutePathFrom(_runOptions.ObservedDataFolder, fileInfo.Name);
          fileInfo.CopyTo(copiedObservedDataFilePath, overwrite: true);
 
          return new ObservedDataMapping
          {
             Id = observedDataMapping.Id,
             Type = observedDataMapping.Type,
-            Path = FileHelper.CreateRelativePath(copiedObservedDataFilePath, _runOptions.ReportConfigurationFile, useUnixPathSeparator:true)
+            Path = pathRelativeToOutputFolder(copiedObservedDataFilePath)
          };
       }
 
+
+      private string pathRelativeToOutputFolder(string fullPath) => FileHelper.CreateRelativePath(fullPath, _runOptions.OutputFolder, useUnixPathSeparator: true);
+
+      private string absolutePathFrom(string relatedTo, string relativePath)
+      {
+         var sanitizeRelativePath = relativePath;
+         if(sanitizeRelativePath.StartsWith(Path.DirectorySeparatorChar.ToString()) || sanitizeRelativePath.StartsWith(Path.AltDirectorySeparatorChar.ToString()))
+            sanitizeRelativePath= sanitizeRelativePath.Substring(1);
+
+         return Path.Combine(relatedTo, sanitizeRelativePath);

```

Comment:
Interestingly, if the relative path starts with \ or /, then the Path.Combine barfs badly. So that way either /project/file.json, or project/file.json will work